### PR TITLE
715-yc-Radius

### PR DESCRIPTION
**Pull request by maverickyuwei**

### CODE REVIEW

File: braph2/graph/measures/Radius.m [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+    %   getName                     - returns the name of radius measure.
+    %   getDescription              - returns the description of radius measure.
+    %   getAvailableSettings        - returns the settings available to the class.
+    %   getMeasureFormat            - returns the measure format
+    %   getMeasureScope             - returns the measure scope
+    %   getParametricity            - returns the parametricity of the measure  
+    %   getMeasure                  - returns the radius class.
+    %   getCompatibleGraphList      - returns a list of compatible graphs.
+    %   getCompatibleGraphNumber    - returns the number of compatible graphs.
+    %
```
Comment (annacanal) [inline braph2/graph/measures/Radius.m:22]: no points at the end

File: braph2/graph/measures/Radius.m [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+            % RADIUS = CALCULATE(M) returns the value of the radius of a
+            % graph.
+            %   
+            % See also Measure, Eccentricity, Diameter, GraphBU, GraphWU, MultiplexGraphBU, MultiplexGraphWU.
+            
+            g = m.getGraph();  % graph from measure class
+            L = g.layernumber();
+            
+            if g.is_measure_calculated('Eccentricity')
+                eccentricity = g.getMeasure('Eccentricity').getValue();
```
Comment (annacanal) [inline braph2/graph/measures/Radius.m:53]: eccentricity = g.getMeasureValue('Eccentricity');

File: braph2/graph/measures/test_Radius.m [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+A = [
+    0     .1  .2  .25  0;
+    .125  0   0   0    0;
+    .2    .5  0   .25  0;
+    .125  10  0   0    0;
+    0     0   0   0    0
+    ];
+    
+known_radius_subgraphs = {[0]};
+
```
Comment (annacanal) [inline braph2/graph/measures/test_Radius.m:13]: Similarly to test_Diameter, dont need '[ ]'
known_radius_subgraphs = {0};

**Review by annacanal — CHANGES_REQUESTED** [review]

File: braph2/graph/measures/Radius.m (hunk 1 above)
Comment (maverickyuwei) [reply braph2/graph/measures/Radius.m]: e6ae4b0

File: braph2/graph/measures/Radius.m (hunk 2 above)
Comment (maverickyuwei) [reply braph2/graph/measures/Radius.m]: 453a4c6

File: braph2/graph/measures/test_Radius.m (hunk 3 above)
Comment (maverickyuwei) [reply braph2/graph/measures/test_Radius.m]: 3f7a022

**Review by annacanal — APPROVED** [review]

**Review by giovannivolpe — APPROVED** [review]

**Pull request merged**